pkgs/kde/plasma/plasma-integration/default.nix

```diff
@@ -54,7 +55,7 @@ mkKdeDerivation {
     "-DKF5XmlGui_DIR=${libsForQt5.kxmlgui.dev}/lib/cmake/KF5XmlGui"
   ];
 
-  extraBuildInputs = [qtwayland xorg.libXcursor];
+  extraBuildInputs = [qtwayland qqc2-breeze-style xorg.libXcursor];
```

Comment:
It's QML so should be propagated.

Reply:
with or without, it doesn't fix the issue.